lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -55,6 +59,7 @@ def compute(self, student, course_structure, scores_client, submissions_scores):
             # self.scores may hold outdated data, force it to refresh on next access
             lazy.invalidate(self, 'scores')
```

Comment:
Nit: For forward compatibility, we should simply invalidate all lazy attributes at the top (not middle) of this method.  Just in case additional code on top later requires any of the lazy attributes.